app/lib/theme/ReadOnlyAdditionalFundingSourcesArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,41 @@
+import { ArrayFieldTemplateProps } from "@rjsf/core";
+
+const ReadOnlyAdditionalFundingSourcesArrayFieldTemplate = (
+  props: ArrayFieldTemplateProps
+) => {
+  return (
+    <>
```

Comment:
suggestion: I've played with this a little and by adding `div` here and removing `additionalFundingSourceWithNumber` and `additionalFundingSourceSection` classes got a better layout.

Reply:
Not sure exactly what you mean, would you mind pasting your code?

Reply:
```
const ReadOnlyAdditionalFundingSourcesArrayFieldTemplate = (
  props: ArrayFieldTemplateProps
) => {
  return (
    <div>
      {props.items.map((item, i) => {
        return (
          <div key={item.key} className="additionalFundingSourceWithNumber">
            <div className="sourceTitle">Additional Funding Source {i + 1}</div>
            <div key={item.key} className="additionalFundingSourceSection">
              <div className="additionalFundingSourceForm">{item.children}</div>
            </div>
          </div>
        );
      })}
      <style jsx>{`
        :global(.additionalFundingSourceForm) {
          flex-grow: 1;
          margin-right: 1rem;
        }
        .sourceTitle {
          font-weight: bold;
          margin: 1em 0 1em 0;
        }
      `}</style>
    </div>
  );
};